src/settings.cpp

```diff
@@ -82,6 +82,8 @@ std::string path_output;
 std::string path_particle_restart;
 std::string path_sourcepoint;
 std::string path_statepoint;
+std::string empty_string = "";
+char const* path_statepoint_c {empty_string.c_str()};
```

Comment:
You don't actually need the empty_string here. path_statepoint is already an empty string, so you can just use that one's c_str.